openapic/compiler.go

```diff
@@ -33,14 +34,33 @@ import (
 )
 
 type PluginCall struct {
-	Name   string
-	Output string
+	Name       string
+	Invocation string
 }
 
 func (pluginCall *PluginCall) perform(document *openapi_v2.Document, sourceName string) {
 	if pluginCall.Name != "" {
 		request := &plugins.PluginRequest{}
-		request.Parameter = ""
+
+		request.Invocation = pluginCall.Invocation
+		invocationParts := strings.Split(pluginCall.Invocation, ":")
+		var outputLocation string
+		switch len(invocationParts) {
+		case 1:
+			outputLocation = invocationParts[0]
+		case 2:
+			parameters := strings.Split(invocationParts[0], ",")
```

Comment:
Added in [this commit](https://github.com/googleapis/openapi-compiler/pull/23/commits/85f46b8beea70901db4dd08066abadcd8a74a070) along with some tests to verify validation.